lerobot/configs/env/aloha.yaml

```diff
@@ -5,10 +5,13 @@ fps: 50
 env:
   name: aloha
   task: AlohaInsertion-v0
-  from_pixels: True
-  pixels_only: False
   image_size: [3, 480, 640]
-  episode_length: 400
-  fps: ${fps}
   state_dim: 14
   action_dim: 14
+  fps: ${fps}
+  episode_length: 400
+  gym:
+    obs_type: pixels_agent_pos
+    render_mode: rgb_array
+    visualization_width: 384
+    visualization_height: 384
```

Comment:
nit: I know we've settled on this design before but now that I see them side by side, wouldn't it be better to have `visualization_size: [384, 384]` for better uniformity? 

Also, why would the `image_size` not go under `gym`?

Reply:
I would prefer `image_size: [3, 480, 640]` to be `image_chw_size: [3, 480, 640]` to be explicit.
We could do `visualization_hw_size: [384, 384]`. But let's not change to much stuff in this PR since we would need to change all gym_env

Reply:
I just removed image_size. It was unused